SSW.CleanArchitecture.sln

```diff
@@ -39,6 +37,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tools", "tools", "{7AFFF91C
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Database", "tools\Database\Database.csproj", "{08A4FCE5-D49A-4FC1-BB0B-3206326157BD}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AppHost", "tools\AppHost\AppHost.csproj", "{874CCCC1-3F73-4DA9-BA2B-89EE6C7C4B8E}"
```

Comment:
I'd consider naming this Aspire.

Reply:
As per our conversation, your comment makes sense, but most of the examples I've seen have this left as the 'AppHost'.  I think keeping it named like aligns with other projects I've seen.